### ISSUE TITLE
Is Time Bonus only considered for perfect runs or also for imperfect runs?

### ISSUE THREAD

**Issue by gokulkchenchani** [robocup-at-work/rulebook#92]
![Screenshot from 2024-07-09 10-49-00](https://github.com/robocup-at-work/rulebook/assets/81406273/021c0fa4-b038-4945-97c1-39e887eb5452)

In the above rule from RuleBook for Time Bonus, does the run have to be a perfect run or it can include penalties? As the rule here just states finishing of the task and does not mention anything about perfect run or an imperfect run.

**Comment by AlcatrAAz** [comment]
AFAIK it should be perfect run. Yes should be more precise.

**Comment by marcomasa** [comment]
Yes, only if a perfect run was achieved before. We need to add it.
